trimmer_decorator.rb

```diff
@@ -0,0 +1,10 @@
+require './decorator'
+
+class TrimmerDecorator < Decorator
+  MAXIMUM_NAME_LENGTH = 10
+
+  def correct_name
+    name = @nameable.correct_name
+    name.length > MAXIMUM_NAME_LENGTH ? name[0..MAXIMUM_NAME_LENGTH] : name
+  end
+end
```

Comment:
as required your function should trimmer inputted words to the length of 10 but while testing I discovered that your function trimmer does not trim 11 letter characters because of the function below.

`
name[0..MAXIMUM_NAME_LENGTH]
`
since the maximum length is = 10 the if statement read the spread operator as 0,1,2,3,4,5,6,7,8,9,10 as 11 characters please changes this to `[0...9]` to fx this error and try again 😄

Reply:
@alexander16108 Thank you for that. I'm on it